### DISCUSSION TITLE
Is it possible to change the icon library? Astro-icon isn't importing icons from other icon sets

### DISCUSSION THREAD

**Discussion by ghost** [arthelokyo/astrowind#321] [Q&A]

**Comment by prototypa** [comment]
Hello @agi-omar. We are moving this to Discussion because it is not an issue with **AstroWind**.

**Reply by ghost** [reply]
Sorry, I thought there was a problem with the configuration. Still not sure about changine the icon set via iconify. Do you have any idea where I can start?
